deploy-sandpit.sh

```diff
@@ -41,13 +42,15 @@ BUILD=0
 OIDC=0
 RECEIPTS=0
 SHARED=0
+UTILS=0
 CLEAN=""
 TERRAFORM_OPTS="-auto-approve"
 if [[ $# == 0 ]]; then
   AM=1
   BUILD=1
   OIDC=1
   SHARED=1
+  UTILS=1
```

Comment:
You haven't stated that this would be the `(default)` behaviour above.

My gut feel is that it shouldn't be default, as its not core to application.